src/js/components/DataTable/ExpanderCell.js

```diff
@@ -31,7 +31,14 @@ const ExpanderControl = ({ context, expanded, onToggle, pad, ...rest }) => {
   delete normalizedThemeProps.pad;
 
   content = (
-    <Box {...normalizedThemeProps} {...rest} align="center" fill pad={pad}>
+    <Box
+      {...normalizedThemeProps}
+      {...rest}
+      align="center"
+      fill
+      pad={pad}
+      justify="center"
```

Comment:
I think there are cases where it is desirable to not have the expander icon vertically centered. For example if you have a DataTable with rows that take up a lot of vertical space, having the expander icon at the top of the row makes it easier to associate with the row (see screenshots below)

Expander icon with `justify="center"`
<img width="741" alt="Screen Shot 2023-01-10 at 3 48 20 PM" src="https://user-images.githubusercontent.com/54560994/211680082-9de79372-e8dd-4ef4-ae97-40db31293f25.png">


Expander icon without `justify="center"`
<img width="734" alt="Screen Shot 2023-01-10 at 3 48 33 PM" src="https://user-images.githubusercontent.com/54560994/211679943-72343895-37fe-47c1-a48d-784d7381043c.png">

I understand a use case like this might not always be the case, but I'm just hesitating to make `justify="center"` the default.

Another note on this. It looks like `verticalAlign` is being passed through but isn't having an effect on the expander cell. This could be something to look into. Perhaps instead of making `justify="center"` the default we can utilize `verticalAlign` so that people can control this?

Reply:
Thanks for your feedback! Those screenshots definitively make it look like a poor idea! 🙈
I'll take a look into `verticalAlign`.

Reply:
Yeah I think a theme setting would make sense. Maybe `dataTable.expand` that can accept button props?